### DISCUSSION TITLE
Qiitaの検索では、検索文字列が抜粋本文の文末にくることが多く、要旨が分かりにくくて不便です

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#465] [要望・アイデア・不具合]

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご不便をおかけして申し訳ありません。
調査したところ、一致箇所の後ろも表示することが可能ですが、
場合によっては一致箇所より後ろが取得できない場合もありそうです。

可能なケースについては検討いたします！
ただ、直近の開発優先度などから考えると先になってしまいそうなので、気長にお待ちいただけると幸いです🙇💦
